library/network/src/modules/SuSEFirewall.rb

```diff
@@ -655,6 +655,29 @@ def morph_to(backend_sym)
       Yast.const_set('SuSEFirewall', self.class.superclass.create(backend_sym))
     end
 
+    # Local function for increasing the verbosity level.
```

Comment:
But the only user of `@verbose_level` is SuSEFirewall2::GetZoneOfInterface so shouldn't the methods stay in the old class?